10-kitten/Containerfile

```diff
@@ -1,27 +1,24 @@
-FROM docker.io/library/almalinux:10-kitten as repos
-FROM quay.io/centos-bootc/bootc-image-builder:latest as builder
+FROM quay.io/almalinuxorg/10-kitten-base:10-kitten as repos
```

Comment:
This is a weird chicken-and-egg problem, isn't it? How is the `10` image going to be built?

(I realize the original also had this issue, and the question probably is for @andrewlukoshko or @yuravk)

Reply:
All of the images depend on the CentOS Stream 10 bootc image for imagectl. 

Each bootc image variant depends on the almalinux repositories, which we're getting from the non-bootc images. The 10-kitten bootc image gets the repo config from the 10-kitten non-bootc image, and we're ready to do the same with the stable AlmaLinux 10 image as soon as the non-bootc image is published.